ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerwidget.js

```diff
@@ -510,7 +510,8 @@ if (typeof window.DatePickerWidget === 'undefined') {
       if (!DatePickerWidget.#visible) {
         let date,
             validDate;
-        validDate = this.#curInstance.selectedDate;
+        
+        validDate = this.#curInstance.selectedDate || this.#model.value;
```

Comment:
Why is this change part of this PR ? Please take care of such things before creating a PR.